src/android/ClickReceiver.java

```diff
@@ -22,7 +22,7 @@
 package de.appplant.cordova.plugin.localnotification;
 
 import android.os.Bundle;
-import android.support.v4.app.RemoteInput;
+import androidx.core.app.RemoteInput;
```

Comment:
This is probably ok, but did we intend to change this import? I know this was probably something that Android Studio did automatically, as it has happened to me before.

Reply:
Yes, indeed. Android Studio did this without my command. I did not intend to do this.